slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -646,6 +691,8 @@ void SLAMServiceImpl::RunSLAM() {
                 latest_map_filename.find(viam::io::filename_prefix) +
                     viam::io::filename_prefix.length(),
                 latest_map_filename.find(".pbstream")));
+
+        localization_map_ready = CacheMapInLocalizationMode();
```

Comment:
[minor]
```suggestion
        CacheMapInLocalizationMode();
```

Reply:
was already making changes so I added this to my commit